CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to the sdc11073 module will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [current]
```

Comment:
```suggestion
## [Unreleased]
```

Reply:
I think release state has nothing to do with the repo itself. I suggest [master], but [current] is also ok for me.

Reply:
done